### DISCUSSION TITLE
Model for 140_Ultra-Fast-Lane-Detection

### DISCUSSION THREAD

**Discussion by Paras1291** [PINTO0309/PINTO_model_zoo#191] [Q&A]
I dont know how to use those codes
can you provide me with th tflite model of 140_Ultra-Fast-Lane-Detection linked to 
(https://github.com/ibaiGorordo/TfLite-Ultra-Fast-Lane-Detection-Inference)

**Comment by PINTO0309** [comment]
https://github.com/PINTO0309/PINTO_model_zoo/tree/main/140_Ultra-Fast-Lane-Detection

**Reply by Paras1291** [reply]
How to extract the tflite model specificaly?
I am just a beginner

**Reply by PINTO0309** [reply]
```bash
sudo chmod +x *.sh
./download_culane.sh
./download_tusimple.sh
```

**Reply by Paras1291** [reply]
Actually i dont use linux, Can you please give me the file directly?

**Reply by PINTO0309** [reply]
1. open a file
![image](https://user-images.githubusercontent.com/33194443/155843509-ec586282-0bc1-4024-9e1a-bf7385ae46f5.png)
2. Open a browser
3. Copy the URL
![image](https://user-images.githubusercontent.com/33194443/155843545-1d091877-b246-4151-8582-9cd738c51c6d.png)
4. Paste the copied information into the URL field of your browser.
![image](https://user-images.githubusercontent.com/33194443/155843577-606cbcc4-80fb-47e7-8005-667f4bc90ef1.png)
5. Press the Enter key.
6. Click "Download anyway".
![image](https://user-images.githubusercontent.com/33194443/155843618-130d545f-c4a1-4e90-b5c4-6a4f7b0c8b68.png)
7. The file will be downloaded.
![image](https://user-images.githubusercontent.com/33194443/155843656-a8fd2816-4c08-4043-8dd6-437d213db699.png)
8. I have no idea what OS you are using, but unzip the tar.gz file as you like.

**Reply by Paras1291** [reply]
these files are too large isnt there a link only for the tflite model?

**Reply by PINTO0309** [reply]
No, there is not. I don't have any plans to create a link.

**Reply by Paras1291** [reply]
is there anyone else who has already done this and has the model?

**Reply by AbdulFMS** [reply]
> is there anyone else who has already done this and has the model?

UFLD Culane:
https://drive.google.com/drive/folders/1-gY6hQ78wbB0njntd0P6uQXhyMtUTYtG?usp=sharing
UFLD Tusimple:
https://drive.google.com/drive/folders/1k9Gusqfwxogwisb_g8Rggb2u-r7--jkK?usp=sharing

**Reply by AbdulFMS** [reply]
here you go champ! @Paras1291

**Reply by Paras1291** [reply]
Thanks a ton @AbdulFMS

**Reply by Ben-schlch** [reply]
@AbdulFMS What are all the different models?  Can you explain how you created them?
